draft-ietf-quic-http.md

```diff
@@ -312,11 +317,14 @@ an explicit port.
 
 ## Connection Establishment {#connection-establishment}
 
-HTTP/3 relies on QUIC as the underlying transport.  The QUIC version being used
-MUST use TLS version 1.3 or greater as its handshake protocol.  HTTP/3 clients
-MUST indicate the target domain name during the TLS handshake. This may be done
-using the Server Name Indication (SNI) {{!RFC6066}} extension to TLS or using
-some other mechanism.
+HTTP/3 relies on QUIC version 1 as the underlying transport.  The use of other
+QUIC transport versions with HTTP/3 MAY be defined by future specifications.
+
+QUIC version 1 uses TLS version 1.3 or greater as its handshake protocol.
+HTTP/3 clients MUST support a mechanism to indicate the target host to the
+server during the TLS handshake.  Unless an alternative mechanism for indicating
+the target host is used, clients MUST use the Server Name Indication (SNI)
+{{!RFC6066}} extension to TLS if the target host is a DNS name.
```

Comment:
According to [RFC 6066](https://tools.ietf.org/html/rfc6066#section-3):
> "HostName" contains the fully qualified DNS hostname of the server,
> as understood by the client.

Should we replace "DNS name" with "fully qualified DNS hostname"?

Reply:
In general, I don't think clients know if the hostname of the server in the URL is fully qualified or not, so the "as understood by the client" part seems important.